publisher.py

```diff
@@ -162,6 +167,7 @@ def parse_arguments(self):
         parser.add_argument("--dry-run", dest="dry_run", action="store_true")
         parser.add_argument("--config", dest="config", default="publisher.yaml")
         parser.add_argument("--filter", dest="images_filter", default=None)
+        parser.add_argument("--retry-limit", dest="retry_limit", default=5)
```

Comment:
Put the default value in a single location.

Reply:
Done.